packages/forms-web-app/src/pages/project-search/utils/filters/get-filters-view-model.js

```diff
@@ -81,7 +81,37 @@ const getFiltersViewModel = (filters) => {
 		);
 	});
 
+	const locationIndex = filterGroups.findIndex((filterGroup) => filterGroup.name === 'region');
+	filterGroups[locationIndex].items = orderLocationItems(filterGroups[locationIndex].items);
+
 	return orderFilterGroups(filterGroups);
 };
 
+const orderLocationItems = (items) => {
+	// north to south order
+	const locationOrder = [
+		'north_west',
+		'north_east',
+		'yorkshire_and_the_humber',
+		'west_midlands',
+		'east_midlands',
+		'eastern',
+		'south_west',
+		'south_east',
+		'london',
+		'wales'
+	];
+	const orderedItems = [];
+	locationOrder.forEach((region) => {
+		const regionIndex = items.findIndex((item) => item.value === region);
+
+		if (regionIndex >= 0) {
+			const regionElement = items[regionIndex];
+			items.splice(regionIndex, 1);
+			orderedItems.push(regionElement);
```

Comment:
just a question for my benefit really, if you are running the loop over a pre-defined array of `locationOrder` where every value appears once, what is the benefit of running `items.splice(regionIndex, 1);` to remove it from items?

Reply:
Good spot - that was from another implementation and thought I had removed it. Will remove now, thanks!

Reply:
Well, it occurred to me later, that removing the item after sorting would make `findIndex()` look through fewer items, which could be a nice optimisation for large arrays, but probably won't do much in this case. You might as well leave it, it's a nice pattern, I just didn't understand it at first!

Reply:
That's a really good point and was totally intentional